### DISCUSSION TITLE
タグの消し忘れ

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#553] [要望・アイデア・不具合]
以下はQiita Zizeの[Advent Calendar](https://zine.qiita.com/tag/advent-calendar/)タグの例ですが、添付の通りにphpの終了タグのようなものが残っていると思われます。Qiita Zizeのタグページをたまたま開いたら、気づきました。

![スクリーンショット 2023-10-23 20 18 12](https://github.com/increments/qiita-discussions/assets/38848514/1a606fd3-a818-4bbe-b428-26db4b73c0b5)

**Comment by tanaka-asami415** [comment]
@Brutus03 
ご報告ありがとうございます！🙏
